net.zgcoder.skycheckers.desktop

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env xdg-open
+
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=false
+Exec=skycheckers
+Name=SkyCheckers
+Comment=Blast enemies off a stage
+Icon=net.zgcoder.skycheckers
+Categories=Game;
+
```

Comment:
The desktop file & the appdata should go upstream if possible.

Reply:
What does upstream mean here? I'm committing them here, and they are in the skycheckers GitHub repo.

Reply:
Well, if they are already in the upstream repository (skycheckers) you should remove them from here

Reply:
Removed desktop and metainfo file from here.